Source/Common/Config.h

```diff
@@ -40,7 +40,7 @@ struct PortableInformation {
  *
  * @return The application name and path structure
  */
-ApplicationNames GetApplicationNames(fextl::vector<fextl::string> Args, bool ExecFDInterp, int ProgramFDFromEnv);
+ApplicationNames GetApplicationNames(const fextl::vector<fextl::string>& Args, bool ExecFDInterp, int ProgramFDFromEnv);
```

Comment:
This `const` is misleading: The container itself isn't modified, but the data is being moved from it.

Reply:
Oop, forgot to remove the move, we actually want the copy since `RecoverGuestProgramFilename` mutates it.
Resolved.